ManoManoTech/firefighter-incident
# feat(slack): invite specific Slack user groups for P1 incidents

# feat(slack): invite specific Slack user groups for P1 incidents 

## Description
Need to add a case when an incident is P1, we have to invite automatically a specific groups of user


## Changes Made
* Add tag field for UserGroup model
* Make a migration to apply this field
* Add a new receiver to add specific group when an incident is P1
* Reorganize the back office presentation of a user groups to be more ux/ui friendly 
* Add a comment into Group model to be more clear. (linked comment for docs)


## How to Test
Create a user group with you and others people, create an P1 incident, and check if all users of this groups are invited to this incident.


## Checklist
